platforms/chibios/_wait.h

```diff
@@ -30,6 +30,11 @@
 
 #ifdef WAIT_US_TIMER
 void wait_us(uint16_t duration);
+#elif PORT_SUPPORTS_RT == TRUE
+#    define wait_us(us)                                            \
+        do {                                                       \
+            chSysPolledDelayX(US2RTC(REALTIME_COUNTER_CLOCK, us)); \
```

Comment:
Does `chSysPolledDelayX` interpret zero as infinite?

Reply:
Nope it just skips the wait http://chibiforge.org/doc/21.11/full_rm/chsys_8c_source.html#l00492